docs/design/profiles.md

```diff
@@ -1,3 +1,18 @@
+---
+title: "Profiles"
```

Comment:
All of this we plan to add in `mdox` - this is just not needed boilerplate https://github.com/bwplotka/mdox

cc @saswatamcode for your context. Observatorium, Thanos and [Prometheus Operator ](https://github.com/prometheus-operator/prometheus-operator) looks like could enormously be improved thanks to our work! Looking forward to it!

Reply:
Sure! Excited to work on this!